content/doc/pipeline/index.adoc

```diff
@@ -1,981 +1,437 @@
 
-```groovy
-step([$class: 'JavadocArchiver', javadocDir: 'target/resources/javadoc', keepAll: false])
-```
+Pipeline-related plugins other than those listed above are regularly “whitelisted” as compatible with or designed for Pipeline usage. For more information, see the link:https://github.com/jenkinsci/workflow-plugin/blob/541faf611659e1e6b8f2cbbd3435756b27633db4/COMPATIBILITY.md[Plugin Compatibility With Pipeline] wiki. Current contents of that wiki are summarized in the following table:
 
-```groovy
-step([$class: 'Fingerprinter', targets: 'target/api.war'])
-```
+SCMs::
+    * GitSCM (git)
+    * SubversionSCM (subversion)
+    * MercurialSCM (mercurial)
+    * PerforceScm (p4, not the older perforce)
+    * IntegritySCM (integrity-plugin)
+    * RepoScm (repo)
+    * teamconcert
+    * CVSSCM (cvs)
+
+Build Steps and Post-Build Actions::
+    * ArtifactArchiver (core)
+    * Fingerprinter (core)
+    * JUnitResultArchiver (junit)
+    * JavadocArchiver (javadoc)
+    * Mailer (mailer)
+    * CopyArtifact (copyartifact)
+    * Analysis publishers
+    * HtmlPublisher (htmlpublisher)
+    * HipChatNotifier (hipchat)
+    * LogParserPublisher (log-parser)
+    * SeleniumHtmlReportPublisher (seleniumhtmlreport)
+    * ScoveragePublisher (scoverage)
+    * AnsiblePlaybookBuilder (ansible)
+    * GitHubCommitNotifier, GitHubSetCommitStatusBuilder (github)
+    * XUnitPublisher and XUnitBuilder (xunit)
+
+Build Wrappers::
+    * ConfigFileBuildWrapper (config-file-provider)
+    * Xvnc (xvnc)
+    * BuildUser (build-user-vars)
+    * TimestamperBuildWrapper (timestamper)
+    * MaskPasswordsBuildWrapper (mask-passwords)
+    * XvfbBuildWrapper (xvfb)
+    * GCloudBuildWrapper (gcloud-sdk)
+    * NpmPackagesBuildWrapper (nodejs)
+    * AnsiColorBuildWrapper (ansicolor)
+
+Triggers::
+    * gerrit-trigger
+    * github
+    * deployment-notification
+    * gitlab-plugin
+    * bitbucket
+
+Clouds::
+    * mock-slave (for prototyping)
+    * docker
+    * Nectar-vmware
+    * operations-center-cloud
+    * ec2
+
+Miscellaneous::
+    * rebuild
+    * parameterized-trigger
+    * build-token-root
+    * job-dsl
+    * zentimestamp
+    * claim
+    * listSubversionTagsParameterValue
+    * authorize-project
+    * customize-build-now
+    * embeddable-build-status
+    * groovy-postbuild
+    * buildtriggerbadge
+    * build-monitor-plugin
+    * radiatorview
+
+Custom Steps::
+    * docker-workflow
+    * credentials-binding
+    * ssh-agen
+    * parallel-test-executor
+    * mailer
+    * email-ext
+
+
+
+*Note:* Several plugins available in the Jenkins ecosystem but not actually related to the Pipeline feature set described in this guide also use the terms "pipeline" or "DSL" in their names. For example:
```

Comment:
The phrasing here is weird, I had to read it twice to make sure I parsed it correctly. Also, asciidoc has the "NOTE: " formatting which will automatically create a note box, which gets rendered like this:

![asciidoc-note-box](https://cloud.githubusercontent.com/assets/26594/13819101/8746a816-eb55-11e5-960d-4e555d80511b.png)

----

Suggested rephrasing of the sentence:

> Several other Jenkins plugins use the terms "pipeline" or "DSL", but are not related to the Pipeline plugin described in this document, for example: